assets/fonts/roboto_mono/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I'm not sure whether it is required to add the font license to the project, but I did it just to be sure,